core/tx_pool.go

```diff
@@ -1163,6 +1184,11 @@ func (pool *TxPool) demoteUnexecutables() {
 		for _, tx := range list.Forward(nonce) {
 			hash := tx.Hash()
 			logger.Warn().Str("hash", hash.Hex()).Msg("Removed old pending transaction")
+			erroredTxns = append(erroredTxns, types.RPCTransactionError{
+				TxHashID:             hash.Hex(),
+				TimestampOfRejection: time.Now().Unix(),
+				ErrMessage:           "old pending transaction",
```

Comment:
more informative message be can returned. Just adding the nonces.

Reply:
Added informative msgs in latest